docs/var.tfvars-doc.md

```diff
@@ -119,7 +120,7 @@ Please note that only OpenSSH formatted keys are supported. Refer to the followi
 Create the SSH key-pair and keep it under the `data` directory
 
 These set of variables specify the RHEL subscription details.
-Update the variables with your subscription id and password.
+They are here just to keep all Terraform variables in use in one place, and because they are secrets, you should not set your values and save it on disk. Instead, set environment variables `RHEL_SUBS_USERNAME` and `RHEL_SUBS_PASSWORD` and pass them to `terraform apply` command as shown in the [Quickstart guide](./quickstart.md).
```

Comment:
""This is sensitive data and if you don't want to save it on disk, use environment variables and pass it to `terraform apply` command as shown in .."

Reply:
Yep, will be in the next push.